### DISCUSSION TITLE
Support text selectable on macOS?

### DISCUSSION THREAD

**Discussion by isaced** [gonzalezreal/swift-markdown-ui#254] [Q&A]
I use `swift-markdown-ui` to render some article content on **macOS**, and I hope to allow users to copy text, how can I support **selectable**?

**Comment by lidian6864677** [comment]
Hello, have you found a solution for this problem

**Comment by laike9m** [comment]
Just use
```swift
Markdown(...)
      .textSelection(.enabled)
```
